tests/unittest_brain.py

```diff
@@ -1612,7 +1612,7 @@ def test_typing_namedtuple_dont_crash_on_no_fields(self):
         self.assertIsInstance(inferred, astroid.Instance)
 
     @test_utils.require_version("3.8")
-    def test_typedDict(self):
+    def test_typed_dict(self):
```

Comment:
`typedDict` would be more fitting

Reply:
I think pep8 says that method name should be in snake case.